boards/px4/fmu-v6xrt/init/rc.board_mavlink

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
 #
-# board specific MAVLink startup script.
+# PX4 FMUv6X-RT specific board MAVLink startup script.
 #------------------------------------------------------------------------------
 
-# Start MAVLink on the UART connected to the mission computer
-mavlink start -d /dev/ttyS5 -b 3000000 -r 290000 -m onboard_low_bandwidth -x -z
+# if skynode base board is detected start Mavlink on Telem2
+if ver hwtypecmp V6X009010 V6X010010
```

Comment:
Do we want to start on SN base with V6XRT? IF so we need to add that VER_REV to the list.

Reply:
For now we want to make sure non-SN base can use the Telem2

Getting it to work on SN would be nice as well though.

Could you handle the VER_REV list I'm not familiar with that mechanism?